c_src/quicer_config.c

```diff
@@ -571,8 +571,73 @@ encode_parm_to_eterm(ErlNifEnv *env,
 {
   ERL_NIF_TERM res = ERROR_TUPLE_2(ATOM_ERROR_NOT_FOUND);
   if (QUICER_PARAM_HANDLE_TYPE_CONN == Type
-      && QUIC_PARAM_CONN_STATISTICS == Param
-      && sizeof(QUIC_STATISTICS) == BufferLength)
+      && QUIC_PARAM_CONN_STATISTICS_V2 == Param
+      && sizeof(QUIC_STATISTICS_V2) == BufferLength)
+    {
+      QUIC_STATISTICS_V2 *statics = (QUIC_STATISTICS_V2 *)Buffer;
+      ERL_NIF_TERM term = enif_make_list(
+          env,
+          36,
+          STAT_TUPLE_U64(ATOM_QUIC_CORRELATION_ID, CorrelationId),
+          STAT_TUPLE_U32(ATOM_QUIC_VERSION_NEGOTIATION, VersionNegotiation),
+          STAT_TUPLE_U32(ATOM_QUIC_STATELESS_RETRY, StatelessRetry),
+          STAT_TUPLE_U32(ATOM_QUIC_RESUMPTION_ATTEMPTED, ResumptionAttempted),
+          STAT_TUPLE_U32(ATOM_QUIC_RESUMPTION_SUCCEEDED, ResumptionSucceeded),
+          STAT_TUPLE_U32(ATOM_QUIC_GREASE_BIT_NEGOTIATED, GreaseBitNegotiated),
+          STAT_TUPLE_U32(ATOM_QUIC_ECN_CAPABLE, EcnCapable),
+          STAT_TUPLE_U32(ATOM_QUIC_ENCRYPTION_OFFLOADED, EncryptionOffloaded),
+          STAT_TUPLE_U32(ATOM_QUIC_RESERVED, RESERVED),
```

Comment:
Q: this sounds like it's not a statistic?

Reply:
probably not, but it is in the struct, I think it is no harm to have it here.

Reply:
double-check it is reserved bits, so it should be fine.